frontend/src/lib/environment-provider.tsx

```diff
@@ -59,3 +70,39 @@ export const useUserPreferences = () => {
         return api.getPreferences()
     })
 }
+
+export const host = () => {
+    const env = useEnvironment()
+    if (env.accountsEnvName === 'production') {
+        return `https://openstax.org`;
+    }
+    return `https://${env.accountsEnvName}.openstax.org`;
+}
```

Comment:
this is actually returning the `origin`, host lacks the scheme.  maybe rename it to be `locationOrigin`?

Reply:
oh yeah this was just moved so i didn't actually 'change' anything here, happy to change this to be more clear though :D